src/backend/catalog/pipeline_query.c

```diff
@@ -561,6 +564,25 @@ IsSWContView(RangeVar *name)
 	return sw;
 }
 
+/*
+ * IsTTLContView
+ */
+bool
+IsTTLContView(RangeVar *name)
+{
+	bool ttl = false;
+	HeapTuple tuple = GetPipelineQueryTuple(name);
+
+	if (HeapTupleIsValid(tuple))
+	{
+		Form_pipeline_query row = (Form_pipeline_query) GETSTRUCT(tuple);
+		ttl = AttributeNumberIsValid(row->sw_attno) || AttributeNumberIsValid(row->ttl_attno);
```

Comment:
Actually, I think we shouldn't even have `sw_attno` now. It should just be a boolean column called `sw`. SW views should simply be TTL views + filtering/combining on read path.